proto/protovalidate-testing/buf/validate/conformance/cases/oneofs.proto

```diff
@@ -48,6 +48,15 @@ message OneofRequired {
   }
 }
 
+message OneofRequiredWithRequiredField {
+  oneof o {
+    option (buf.validate.oneof).required = true;
+
+    string a = 1 [(buf.validate.field).required = true];
```

Comment:
Aside: this seems like a thing to lint for, no? This effectively makes it impossible to ever use field `b` and the message be valid, right?

Reply:
Yep, I only included it to be exhaustive! I believe @oliversun9 is covering that in the linting work.

Reply:
Yep, this is covered in https://github.com/bufbuild/buf/pull/2528